Dockerfile

```diff
@@ -21,16 +21,17 @@ RUN apt-get update \
     libzmq3-dev \
     libdb4.8-dev \
     libdb4.8++-dev \
-    git
-RUN git clone https://github.com/bitcoin/bitcoin.git \
-  && cd bitcoin \
-  && git checkout b641f60425674d737d77abd8c49929d953ea4154 \
+    git \
+    curl
+RUN curl -Lo bitcoin.tar.gz https://github.com/bitcoin/bitcoin/archive/v0.17.0rc1.tar.gz \
```

Comment:
This still compiles, I'd rather just fetch the binary tarball and use that. You can see an example of how to fetch it, verify it and install it here:

https://github.com/ElementsProject/lightning/blob/8ff4418faaeba46e222fe8b65879f54e44ebfc17/Dockerfile#L24-L39